### DISCUSSION TITLE
Target Closed whilst launching in one machine only.

### DISCUSSION THREAD

**Discussion by pingu2k4** [hardkoded/puppeteer-sharp#2106] [Q&A]
### Description

I have an application being run by clients, and am using Puppeteer Sharp within it.
Everything is working for everyone, apart from one person who is having a bug crash the application at the point where we are launching the browser.
I am getting a TargetClosedException whilst launching, stating Protocol Error. Target Closed.

I have foudn various similar looking issues but nothing which helps me fix my situation.

### Complete minimal example reproducing the issue

The exception is being thrown at the final line in this code, LaunchAsync.
```csharp
            var options = new BrowserFetcherOptions()
            {
                Path = Path.Combine(FileSystem.Current.AppDataDirectory, ".local-chromium"),
                Product = Product.Chrome
            };

            var browserFetcher = new BrowserFetcher(options);
            await browserFetcher.DownloadAsync();

            var userDataDir = Path.Combine(FileSystem.Current.AppDataDirectory, ".chromium-user-data");

            if (!Directory.Exists(userDataDir))
            {
                Directory.CreateDirectory(userDataDir);
            }

            var rev = await browserFetcher.GetRevisionInfoAsync();

            var launchOptions = new LaunchOptions
            {
                Headless = false,
                Args = new[] { "--window-size=500,500" },
                UserDataDir = userDataDir,
                ExecutablePath = rev.ExecutablePath
            };

            _browser = await Puppeteer.LaunchAsync(launchOptions);
```

### Expected behavior:

Launches the browser for everyone without exceptions thrown

### Actual behavior:

For one person, the following errors (apologies for only having screenshots)
Outer Exception (ProcessException). Message: Failed to create connection
Stack Trace:
![image](https://user-images.githubusercontent.com/11229848/226212073-684df6dd-ec8e-4f40-8781-40f5e3cba832.png)

Inner Exception (TargetClosedException). Message: Protocol error(Target.setAutoAttach): Target closed. (The remote party closed the WebSocket connection without completing the close handshake.)
Stack Trace:
![image](https://user-images.githubusercontent.com/11229848/226212134-f264b05c-19d1-4e66-8b71-7b84de13c37f.png)

### Versions

PuppeteerSharp 9.0.2
.NET 7.0.4

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by kblok** [comment]
I am moving this to a discussion. Let's create an issue when you have something I can test locally.
